rustls-platform-verifier/src/verification/mod.rs

```diff
@@ -74,14 +74,17 @@ fn invalid_certificate(reason: impl Into<String>) -> rustls::Error {
     )))
 }
 
-#[cfg(any(windows, target_os = "android"))]
 /// List of EKUs that one or more of that *must* be in the end-entity certificate.
 ///
 /// Legacy server-gated crypto OIDs are assumed to no longer be in use.
 ///
 /// Currently supported:
 /// - id-kp-serverAuth
 // TODO: Chromium also allows for `OID_ANY_EKU` on Android.
+#[cfg(target_os = "windows")]
```

Comment:
If we have to declare these separately, maybe it makes more sense to move them into `android.rs` and `windows.rs`?

Also, maybe this is worth two lines of comments explaining the context here and/or linking to the issue?

Reply:
I would prefer to keep these next to each other in the source since we still believe they should have the OIDs inside in sync for consistency between platforms.

Reply:
I've updated this code to include a link to the issue and context around why the string handling looks the way it does.